python/tvm/relay/build_module.py

```diff
@@ -249,6 +250,10 @@ def get_params(self):
             ret[key] = value.data
         return ret
 
+    def get_irmodule(self):
+        """Returns the Target IRModule's from code generation"""
```

Comment:
Do we want to say something like per target primitive IRModule that is the input for code generation ?
IRModule from code generation does not sound right. WDYT ?

Reply:
This is the `Map<Target, IRModule>` post-lowering, I can change it to `post-lowering` instead of `for code generation`. I'd rather not prescribe `primitive` though as in a unified world we'd probably not remember to remove this comment.

Reply:
"post-lowering" seems good to me.